tests/contract-playground/src/inheritance/ExtendedInheritance.sol

```diff
@@ -15,6 +15,7 @@ contract ExtendedInheritance is InheritanceBase {
         for (uint256 i = 0; i < 3; i++) {
             target.delegatecall(abi.encodeWithSignature("doSomething(uint256)", i));
         }
+        s_baseAddress = target;
```

Comment:
I've added this to test the case where inheriting contracts use the state variables outside of the constructor. This should not be caught, but it is.